scripts/build.sh

```diff
@@ -9,10 +9,10 @@ if [ -n "$BUILD_VERSION" ]; then
 	VERSION=$BUILD_VERSION
 fi
 
-if [ -z "$USE_RELEASE" ]; then
-	BUILD_VERSION="$VERSION-$BUILD_TIME"
-else
+if [ -z "$NIGHTLY_RELEASE" ]; then
```

Comment:
Doesn't need to be changed because this script isn't setting it, but this would fail under `set -u`... the

```sh
if [ -z "${NIGHTLY_RELEASE:-}" ]; then`
```

form would need to be used to avoid tripping the unbound variable checker.